src/redux/store.js

```diff
@@ -82,4 +82,6 @@ export default createStore(combineReducers(
     rootReducer,
     usersList,
   },
-));
+),
+// eslint-disable-next-line no-underscore-dangle
+window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```

Comment:
why you need this?

Reply:
I had a bug related to redux on logout, found this helpful for debugging redux states and changes, I can remove it

Reply:
done